intro-to-R-tidyverse/03-intro_to_r_exercise.Rmd

```diff
@@ -0,0 +1,197 @@
+---
+title: 'Exercise: Getting comfortable with R'
+output:
+  html_notebook:
+    toc: yes
+    toc_float: yes
+---
+
+The goal of these exercises is to help you get comfortable with using R and R 
+notebooks. 
+
+We will create a play data set for an imaginary mouse experiment where you have 
+control and treatment groups of mice. 
+
+### Create a imaginary mouse experiment
+
+Use the empty chunk below to create a *character* vector with made up sample IDs 
+for your mice in your imaginary experiment.
+Call this new vector `mouse_id` and make sure each ID is distinct.  
+
+```{r}
+mouse_id <-
+```
+
+Check your environment panel. Is `mouse_id` there? 
+
+Use the empty code chunk below to evaluate the structure and type of your object.
+
+```{r}
+
+```
+
+Now evaluate the `length` of `mouse_id`. 
+Remember you can look up information about a function using `?`.
+
+```{r}
+
+```
+
+Now use the `unique` function to check if your `mouse_id` vector has all unique
+ids. 
+Print out the original `mouse_id` vector and the unique `mouse_ids`s and compare 
+them. 
+
+```{r}
+
+```
+
+Now make a *factor* vector that is the same length as `mouse_id` but contains 
+the experiment group information for each mouse.  
+- Call this vector `exp_group`.  
+- Stick to 2 or 3 experimental groups.  
```

Comment:
Here is where it gets a little confusing I believe. Saying that this vector containing the experimental groups should be the same length as `mouse_id` would give off the impression that there should be a designated experimental group per `mouse_id`. Perhaps explicitly stating that there can be more than 1 `mouse_id` per group (or something along those lines) may prevent any clarification issues.

Reply:
I'm not sure I understand. What do you think I'm asking for `exp_group` to be?

Reply:
`mouse_id` is meant to be a unique sample identifier while `exp_group` is meant to be a factor that has perhaps 2 or 3 unique values like "control" "experiment" and designates what group each `mouse_id` belongs to

Reply:
Just made an adjustment. Let me know if that helps.

Reply:
It would be sample groups essentially, correct? i.e. control, then subjects given 2mg versus 10mg of a substance.

Reply:
Yes. Whatever made up experiment groups they want. I don't really care so much about what the groups are so long as there are groups and 2 or three of them.

Reply:
This is just so they have to learn how to evaluate the `levels` of their factor which will become a pertinent thing when we do differential expression in another module.

Reply:
Okay I like the adjustment you made. I like that it clarifies the idea of group membership and also provides an example of an experimental group for further clarification.